engine/apps/api/serializers/channel_filter.py

```diff
@@ -56,22 +57,6 @@ def get_slack_channel(self, obj):
             "id": obj.slack_channel_pk,
         }
 
-    def validate(self, attrs):
```

Comment:
Where jinja template is validated?

Reply:
This was used to ensure that the routes are based on the unique regular expressions. I decided to drop uniqueness, if user wants to create exactly same route which will never get it's alert, let them do that, may be they'll want to modify it later.